src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
+
+export default store;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/203059898-de7c9aa6-8de3-4511-afa3-273aafa0924b.png)

- [ ] You didn't add any create a reducer for categories which is a project requirements. Kindly follow those project requirements which you have missed :+1: shown in the image above :arrow_up:

Reply:
Implemented. Thank you.